service/orchestrator/cloud_service.go

```diff
@@ -58,31 +60,39 @@ func (s *Service) RegisterCloudService(_ context.Context, req *orchestrator.Regi
 	service.Description = req.Service.Description
 
 	// Persist the service in our database
-	s.db.Create(&service)
+	err = s.storage.Create(&service)
```

Comment:
This creates an unnecessary pointer pointer, since service is already `*CloudService`